README.md

```diff
@@ -245,6 +245,42 @@ try {
 }
 ```
 
+### Passwordless Authentication
+
+This library supports [Passwordless Authentication](https://auth0.com/docs/connections/passwordless) to allow users to log in without the need to remember a password.
+
+The email flow supports sending both a code or link to initiate login:
+
+```java
+try {
+    PasswordlessEmailResponse = auth.startPasswordlessEmailFlow("user@domain.com", PasswordlessEmailType.CODE)
+        .execute();
+} catch (Auth0Exception e) {
+        // handle request error
```

Comment:
nit, white-spacing should match the rest

Reply:
Pushed a commit to fix